clusterloader2/pkg/test/interface.go

```diff
@@ -64,3 +66,15 @@ type Executor interface {
 	ExecutePhase(ctx Context, phase *api.Phase) *errors.ErrorList
 	ExecuteObject(ctx Context, object *api.Object, namespace string, replicaIndex int32, operation OperationType) *errors.ErrorList
 }
+
+// TestReporter is an interface for reporting tests results.
+type TestReporter interface {
+	SetTestName(name string)
+	GetTestName() string
+	GetNumberOfFailedSpecs() int
+	GetTestSuiteDescription() string
+	BeginTestSuite()
+	EndTestSuite()
+	ReportTestStepFinish(stepSummary *ginkgotypes.SpecSummary)
```

Comment:
Can we make this interface more abstract and not depend on a specific implementation (`gingotypes.SpecSummary`)?
I believe the `gingotypes.SpecSummary` can be constructed from the following primitives: stepName, duration, error, can't it? It could be done in the implementation of this interface

Reply:
Done.